nova/tests/fixtures/nova.py

```diff
@@ -202,6 +202,21 @@ def delete_stored_logs(self):
         self.logger._output.truncate(0)
 
 
+class DisableSAPQuotaEngineCacheFixture(fixtures.Fixture):
+    class FakeCache:
+        def get(self, key):
+            return None
```

Comment:
IIRC, `oslo.cache` has a special value it returns when things are not in the cache. We use it e.g. [here](https://github.com/sapcc/nova/blob/4767161b98479270c3264bd4806eaf7d320d4b5b/nova/scheduler/mixins.py#L62). I think we should switch to that instead.

Reply:
We might use it there, but `nova.cache_utils.CacheClient` wraps the backend and only hands out `None`, so the current implementation is correct. We could just think about adding a `delete()` so we expose the same interface as `CacheClient`